### PR TITLE
feat: add rpmbuild base images for packaging

### PR DESCRIPTION
**Pull request by BobyMCbobs**
adds the following rpmbuild images
- CentOS 7
- CentOS Stream 8
- CentOS Stream 9

### CODE REVIEW

File: images/rpmbuild-centos-stream9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM ghcr.io/geonet/base-images/centos:stream8
+# Install prerequisites
+RUN dnf module enable -y nodejs:16 \
```
Comment (Mossman1215) [inline images/rpmbuild-centos-stream9/Dockerfile:3]: nodejs 16 is the default for el9 so this line is not needed.

Comment (BobyMCbobs) [reply images/rpmbuild-centos-stream9/Dockerfile:3]: Good catch. Must've pasted in the same as stream8. Updating now

**Comment by BobyMCbobs** [comment]
@Mossman1215, build times are really long for these
https://github.com/GeoNet/base-images/actions/runs/6318670598/job/17158166748?pr=105
looking like 4mins.
I reckon we use larger build machines or not build these images on branches (but that's less helpful)

**Comment by Mossman1215** [comment]
> @Mossman1215, build times are really long for these
> https://github.com/GeoNet/base-images/actions/runs/6318670598/job/17158166748?pr=105
> looking like 4mins.
> I reckon we use larger build machines or not build these images on branches (but that's less helpful)

The latter is my preference.
At the moment these are running every build of packaging per package. So building them here will be saving 100*4 mins total build time in our account which I think will be a decent saving?

**Review by Mossman1215 — APPROVED** [review]
thanks!

**Pull request merged**